### PR TITLE
chore: Update to use build id

### LINKED ISSUE #69: testing: remove random service name suffix in favor of build ID
The limitation of 63 characters for a service name is now removed, switch to the simpler $BUILD_ID.

### PR DESCRIPTION
**Pull request by averikitsch**
Fixes #69

### CODE REVIEW

File: vpc-sample/tests.cloudbuild.yaml [hunk 1]
```diff
@@ -22,9 +22,8 @@ steps:
   entrypoint: /bin/bash
   args:
   - '-c'
-  - | # Cloud Run Service name must be less than 63 characters
-    cat /dev/urandom | LC_CTYPE=C tr -dc 'a-z0-9' | head -c 15 > _short_id # Generate a random 15 character alphanumeric string (lowercase only)
-    gcloud alpha run deploy ${_SERVICE}-$(cat _short_id) \
+  - |
+    gcloud alpha run deploy ${_SERVICE}-$BUILD_ID \
```
Comment (grayside) [inline vpc-sample/tests.cloudbuild.yaml:26]: Likely out of scope, but maybe we don't need alpha here anymore.

File: testing/cloudbuild-templates/cloud-run-template.cloudbuild.yaml [hunk 2]
```diff
@@ -58,7 +57,7 @@ steps:
   - |
     set -x
     gcloud --quiet container images delete gcr.io/${PROJECT_ID}/${_SERVICE}:${SHORT_SHA}
-    gcloud --quiet run services delete ${_SERVICE}-$(cat _short_id) --region ${_SERVICE_REGION} --platform managed
+    gcloud --quiet run services delete ${_SERVICE}-$BUILD_ID --region ${_SERVICE_REGION} --platform managed
```
Comment (grayside) [inline testing/cloudbuild-templates/cloud-run-template.cloudbuild.yaml:60]: Out of scope: `--platform managed` unneeded.

**Review by grayside — APPROVED** [review]

File: testing/cloudbuild-templates/cloud-run-template.cloudbuild.yaml (hunk 2 above)
Comment (averikitsch) [reply testing/cloudbuild-templates/cloud-run-template.cloudbuild.yaml]: updated

File: vpc-sample/tests.cloudbuild.yaml (hunk 1 above)
Comment (averikitsch) [reply vpc-sample/tests.cloudbuild.yaml]: updated

**Pull request merged**